pkg/hooks/audit_log_file.go

```diff
@@ -0,0 +1,45 @@
+package hooks
+
+import "github.com/square/p2/pkg/logging"
+
+// Returns a FileAuditLogger using the given logger
+func NewFileAuditLogger(logger *logging.Logger) *FileAuditLogger {
```

Comment:
probably FileAuditLogger is preferable LOL. i don't have a better suggestion so 👍